packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -109,473 +102,592 @@ const PersonalDetailsForm = ({
-                            <Field name='account_opening_reason'>
+                                ))}
+                            </RadioGroup>
+                        )}
+                        {'first_name' in values && (
+                            <FormInputField
+                                name='first_name'
+                                required={is_svg || is_appstore}
+                                label={getNameAndDobLabels().first_name_label}
+                                hint={getFieldHint(localize('first name'))}
+                                disabled={
+                                    isFieldImmutable('first_name', editable_fields) ||
+                                    (values?.first_name && has_real_account)
+                                }
+                                placeholder={localize('John')}
+                                data-testid='first_name'
+                            />
+                        )}
+                        {'last_name' in values && (
+                            <FormInputField
+                                name='last_name'
+                                required={is_svg || is_appstore}
+                                label={getNameAndDobLabels().last_name_label}
+                                hint={getFieldHint(localize('last name'))}
+                                disabled={
+                                    isFieldImmutable('last_name', editable_fields) ||
+                                    (values?.last_name && has_real_account)
+                                }
+                                placeholder={localize('Doe')}
+                                data-testid='last_name'
+                            />
+                        )}
+                        {!is_appstore && !is_qualified_for_idv && !is_rendered_for_onfido && (
+                            <FormSubHeader title={localize('Other details')} />
+                        )}
+                        {'date_of_birth' in values && (
+                            <DateOfBirthField
+                                name='date_of_birth'
+                                required={is_svg || is_appstore}
+                                label={getNameAndDobLabels().dob_label}
+                                hint={getFieldHint(localize('date of birth'))}
+                                disabled={
+                                    isFieldImmutable('date_of_birth', editable_fields) ||
+                                    (values?.date_of_birth && has_real_account)
+                                }
+                                placeholder={localize('01-07-1999')}
+                                portal_id={is_appstore ? '' : 'modal_root'}
+                                data_testid='date_of_birth'
+                            />
+                        )}
+                        {!is_svg && 'place_of_birth' in values && (
```

Comment:
clarification:
Do we need this !is_svg check? The values will be based on personal-details-config file right?
In config file, place of birth is not supported in svg landing company.

```

 place_of_birth: {
            supported_in: ['maltainvest', 'iom', 'malta'],
            default_value: account_settings.place_of_birth
                ? residence_list.find(item => item.value === account_settings.place_of_birth)?.text
                : '',
            rules: [['req', localize('Place of birth is required.')]],
        },
```

Reply:
We're adding this field for svg as well. It'll be available for both. Need the placement based on this condition.